### DISCUSSION TITLE
Using charges file as charge information in GFN-FF

### DISCUSSION THREAD

**Discussion by epunzal2** [grimme-lab/xtb#610] [Q&A]
The xtb docs say that I can do a GFN-xTB run and use the _charges_ file to constrain the GFN-FF charges. I run GFN2 to get a _charges_ file and then run GFN-FF but it looks like the resulting _gfnff_charges_ file is not affected. What should I do for the program to read the _charges_ file for GFN-FF? I use version 6.4.1, thanks.

**Comment by awvwgk** [comment]
The read in `charges` only affect the assignment of fragment charges. This is mostly relevant for PDB inputs where every residue is considered a fragment and noncovalently bound ion-pairs or charged complexes.
